### DISCUSSION TITLE
Welcome to MobiDAX Discussions!

### DISCUSSION THREAD

**Discussion by alinetskyi** [MobiDAX/opendax#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ibrahimmohammedevuti** [comment]
A quick question, can I add any coIns pairs/ERC-20 token, or it only the coins on the demo allowed.

thank you
